### PR TITLE
Assignment 2 OK

### PR DESCRIPTION
**Pull request by marioruiz2003**

### CODE REVIEW

File: Assignment2/marioruiz2003-MarioRuiz.ttl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+   rdfs:includes :Computer101
+.
+
+:Sensor029 rdfs:hasMeasurement :Measurement8401.
+
+:Computer101 rdfs:hasOwner :User10A;
+   
+.
+:Measurement8401 rdfs:hasTemperature "29";
+     rdfs:atTime "2010-06-12T12:00:12".
```
Comment (ocorcho) [inline Assignment2/marioruiz2003-MarioRuiz.ttl:14]: Wrong use of rdfs prefix, and you should add a datatype to the strings hereand in the previous triple.

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
I was asking for chhanges, but now I realised that you are also deleting some of your other students files. This is not acceptable. Closing this pull request.

**Comment by marioruiz2003** [comment]
I did not know I was deleting other student files. I am not used to using
this platform, while I was trying to upload my answer, in GitHub Desktop I
was asked to pull origin and when I accepted I had downloaded 66 commits
and then I don't know why I have my partner files in my repository, so I
deleted them. I did not know that this would delete it from GitHub.  Why
would I delete my partners files? Please I was not conscious that this
could happen. Would you let me do another pull request? Let me know.

**Comment by dgarijo** [comment]
If you synchronize from origin, then you delete and do a pull request you are in fact proposing to delete existing files.
Oscar did not say you can't do another pull request. Open a new one without deleting the files from your classmates

**Pull request closed without merging**